src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/handlers/presubmit/RecordDecisionAppender.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.hmcts.reform.bailcaseapi.domain.handlers.presubmit;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.bailcaseapi.domain.entities.BailCase;
+import uk.gov.hmcts.reform.bailcaseapi.domain.entities.BailCaseFieldDefinition;
+import uk.gov.hmcts.reform.bailcaseapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.bailcaseapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.bailcaseapi.domain.entities.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.bailcaseapi.domain.entities.ccd.callback.PreSubmitCallbackStage;
+import uk.gov.hmcts.reform.bailcaseapi.domain.handlers.PreSubmitCallbackHandler;
+
+import static java.util.Objects.requireNonNull;
+
+@Slf4j
+@Component
+public class RecordDecisionAppender implements PreSubmitCallbackHandler<BailCase> {
+
+    @Override
+    public boolean canHandle(PreSubmitCallbackStage callbackStage, Callback<BailCase> callback) {
+        requireNonNull(callbackStage, "callbackStage must not be null");
+        requireNonNull(callback, "callback must not be null");
+
+        return callbackStage == PreSubmitCallbackStage.ABOUT_TO_START
+            && callback.getEvent() == Event.RECORD_THE_DECISION;
+    }
+
+    @Override
+    public PreSubmitCallbackResponse<BailCase> handle(PreSubmitCallbackStage callbackStage,
+                                                      Callback<BailCase> callback) {
+
+        if (!canHandle(callbackStage, callback)) {
+            throw new IllegalStateException("Cannot handle callback");
+        }
+
+        final BailCase bailCase = callback.getCaseDetails().getCaseData();
+
+        String conditionAppearance = "The applicant is to appear before an Immigration Officer at [location] between"
+            + " [times] on [date] or any other place and on any other date and time that may be required by the Home"
```

Comment:
@neha-aggarw @ThomasKKC we have been adding the paragraphs in like this, for checkstyle. Is this OK or is there any other better way to do it do you think?

Reply:
If you do formatting using IDE then it will literally put everything in new line, but the new lines can go in the end of previous lines.

Reply:
Ok as far as I know the only checkstyle issue is that the '+' must be on each new line. I saw it in Daniel's IDE and it didn't look that bad